tasks/envs-patch-backward-compatibility.yml

```diff
@@ -0,0 +1,112 @@
+---
+
+- name: "archivematica_src_search_enabled"
+  set_fact:
+    "{{ item.name }}": "{{ item.dict|combine({item.key: archivematica_src_search_enabled}) }}"
```

Comment:
This need an "always" tag, else some vars are not populated, and running other tasks fails.